### PR TITLE
Fixes #80. Default box now doesn't appear in chrome.

### LINKED ISSUE #80: Default box does not appears on Chrome.
When the site loads on Firefox, it looks like this - 
screenshot from 2017-02-19 06-13-47

While on Chrome- it appears just for seconds and get disappear and ends up loading the search bar.

### PR DESCRIPTION
**Pull request by sandeepm96**
Earlier the default box used to appear for like a second when the page just loads. Now it has been fixed. :)

### CODE REVIEW

**Comment by harshit98** [comment]
@sandeepm96 Please include a screenshot of the following fix which has been done :)

**Comment by sandeepm96** [comment]
Screenshot from Chrome:-
screenshot from 2017-02-28 09 47 03

Screenshot from Mozilla:-
screenshot from 2017-02-28 09 47 27

**Comment by sandeepm96** [comment]
@nikhilrayaprolu mentioned that the default box is only supposed to be appearing in Mozilla as it is meant for only Mozilla and it was to be stopped completely from appearing in chrome. So I removed its appearance completely from chrome. :)

**Comment by nikhilrayaprolu** [comment]
@sandeepm96 good work , can you write the css in the component css file instead of adding it as an inline css

**Comment by sandeepm96** [comment]
@nikhilrayaprolu I have replaced inline css in the latest commit. Please review it. :)

**Comment by sandeepm96** [comment]
I have made the home page responsive. The search bar now is responsive and appears well for every screen size. Here are some screenshots:-

Screen width - 360px


Screen width - 480px


Screen width - 768px


Screen width - 1280px


Screen width - 1366px


Couldnot start a new pull request because this was not closed. 
Suggestions are welcome if something needs to be changed. :)

**Comment by sandeepm96** [comment]
I have made the home page responsive. The search bar of home is now responsive and appears well for every screen size.  I am getting some error while uploading the screenshots. I will be uploading them soon. 
Could not start a new pull request as this was not closed.
Suggestions are welcome if something needs to be changed. :)

File: src/app/index/index.component.css [hunk 1]
```diff
@@ -4,5 +4,23 @@
...
 #search-bar{
   position: absolute;
-  left: 300px;
+}
+
+#set-susper-default{
+  display:none;
+}
+
+@media screen and (max-width: 480px) {
```
Comment (nikhilrayaprolu) [inline src/app/index/index.component.css]: @sandeepm96 make a separate pull for the responsive design .dont push into this pull

Comment (nikhilrayaprolu) [inline src/app/index/index.component.css]: have a separate pull per each issue

Comment (sandeepm96) [inline src/app/index/index.component.css]: Ok the third commit is not supposed to be in this PR. How can I discard it from this PR? Help needed.

**Comment by sandeepm96** [comment]
@nikhilrayaprolu I have rolled back to the commit. This PR now doesn't have the responsiveness fix. :)

**Comment by mariobehling** [comment]
Thank you!

**Pull request merged**